lib/emoji.rb

```diff
@@ -63,19 +63,24 @@ def edit_emoji(emoji)
 
   # Public: Find an emoji by its aliased name. Return nil if missing.
   def find_by_alias(name)
-    names_index[name]
+    @mutex.synchronize do
```

Comment:
@mislav Yeah, remove this check - it's a solution. I'm going to make needed correction to this PR.

About populate all emoji to memory - I think it's not great idea, because, sometimes, library haven't been used ever during the program work, sometimes it's one or two call. And, by the way, this `NoMethodError`  - it's a rare problem ( in our program), and re-design the library for this - it's not necessary. It's just my opinion :)